shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -19,38 +19,142 @@ static UIKeyboardType ToUIKeyboardType(NSString* inputType) {
   return UIKeyboardTypeDefault;
 }
 
-@interface FlutterTextInputView : UIView<UIKeyInput>
+#pragma mark - FlutterTextPosition
 
+/** An indexed position in the buffer of a Flutter text editing widget. */
+@interface FlutterTextPosition : UITextPosition
+
+@property(nonatomic, assign) NSUInteger index;
```

Comment:
Was originally read-write; rewrote to be immutable but missed the property decl. Done!